demo/index.html

```diff
@@ -1,81 +1,758 @@
+                                    a draggable element but doesn't start dragging
+                                </p>
+                            </li>
+                            <li>
+                                <p>
+                                    Raise the <span class="snippet">dblclick</span> event when
+                                    there's a new <span class="snippet">touchstart</span> right
+                                    after a click
+                                </p>
+                            </li>
+                            <li>
+                                <p>
+                                    Raise the <span class="snippet">contextmenu</span> event when
+                                    the touch lasts a while but the user doesn't start dragging the
+                                    element.
+                                </p>
+                            </li>
+                        </ul>
+                    </section>
+                    <section id="overriding-default-behaviour">
+                        <h2>
+                            <a href="#overriding-default-behaviour"
+                                >Overriding the default behaviour</a
+                            >
+                        </h2>
+                        <p>
+                            The following options can be passed into the enabling function to change
+                            how the polyfill works:
+                        </p>
+                        <div>
+                            <table>
+                                <thead>
+                                    <tr>
+                                        <th class="attribute">Attribute</th>
+                                        <th class="type">Type</th>
+                                        <th class="description">Description</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <tr>
+                                        <td class="attribute">
+                                            <span class="snippet">allowDragScroll</span>
+                                        </td>
+                                        <td class="type"><span class="snippet">boolean</span></td>
+                                        <td class="description">
+                                            <span>
+                                                Determines whether to allow scrolling when a drag
+                                                reaches the edges of the screen.
+                                            </span>
+                                            <br />
+                                            <span class="default-value">Default: true</span>
+                                        </td>
```

Comment:
There's no real need to tag every column with its role: the `<th>` already cover that. If we need distinct styling, we can use `table { tbody { td { &:nth-child(...) { ... }}}}` without needing to pad the HTML itself.